datalad/config.py

```diff
@@ -49,6 +51,11 @@
           modification. This can be disable to make multiple sequential
           modifications slightly more efficient.""".lstrip()
 
+# Selection of os.stat_result fields we care to collect/compare to judge
+# on either file has changed to warrant reload of configuration.
+# We cannot just take a full record since st_atime could potentially
+# change on every  git config  call.
+_stat_result = namedtuple('_stat_result', 'st_ino st_size st_ctime st_mtime')
```

Comment:
[ just some thoughts; I'm okay with this as is ]

While I like namedtuples and they are generally cheap, I don't see any spot where we actually access the tuple and to my eyes defining this doesn't improve readability over using a plain tuple and having the above comment to `_get_stats`.  (In fact, I think having that comment local in `_get_stats` would be an improvement.)

Reply:
Indeed with removal of use of `.st_mtime` for time comparison there is more sense to just use a tuple. And I kinda both like and dislike namedtuples: indeed adds a bit of clutter and need to keep assignment order in sync with names.  But I did find having a namedtuple (over a regular tuple) **much** better during debugging whenever I simply see what is that `[n]`th element instead of constantly looking up and doing visual matching for an indexed entry.  so I would prefer to keep using namedtuple (did timing -- should be the same for our purposes).

Re comment placement, I have no strong feeling, but typically I prefer description on the purpose of the variable/type by its definition in such cases.  But indeed it might be "too much of information" for this purpose -- I will just strip away 2nd half of the comment (on why not a full one).